cmake/Modules/FindCurl_EP.cmake

```diff
@@ -81,8 +81,8 @@ if (NOT CURL_FOUND AND TILEDB_SUPERBUILD)
       # Set download name to avoid collisions with only the version number in the filename
       DOWNLOAD_NAME ep_curl.tar.gz
       URL
-        "https://github.com/TileDB-Inc/tiledb-deps-mirror/releases/download/2.11-deps/curl-7.74.0.tar.gz"
-        "https://curl.se/download/curl-7.74.0.tar.gz"
+        "https://github.com/TileDB-Inc/tiledb-deps-mirror/releases/download/2.11-deps/curl-7.88.1.tar.gz"
```

Comment:
I think we can remove this one, or we need to create a `2.16-deps` and add the tar.

Reply:
Done